src/Operator/Constructors.jl

```diff
@@ -161,12 +162,112 @@ operator field:
   evaluation mode:
     gradient
 ```
+
+# SUPG mass matrix example:
+```jldoctest
+# setup
+mesh = Mesh2D(1.0, 1.0);
+parameters = (wind = [1, 1]);
+mapping = nothing;
+collocate = false;
+supgmass = GalleryOperator("supgmass", 4, 4, mesh, parameters, collocatedquadrature = collocate, mapping = mapping);
+
+# verify
+println(supgmass)
+
+# output
+
+finite element operator:
+2d mesh:
+    dx: 1.0
+    dy: 1.0
+
+2 inputs:
+operator field:
+  tensor product basis:
+    numbernodes1d: 4
+    numberquadraturepoints1d: 4
+    numbercomponents: 1
+    dimension: 2
+  evaluation mode:
+    interpolation
+operator field:
+  tensor product basis:
+    numbernodes1d: 4
+    numberquadraturepoints1d: 4
+    numbercomponents: 1
+    dimension: 2
+  evaluation mode:
+    quadratureweights
+
+1 output:
+operator field:
+  tensor product basis:
+    numbernodes1d: 4
+    numberquadraturepoints1d: 4
+    numbercomponents: 1
+    dimension: 2
+  evaluation modes:
+    interpolation
+    gradient
+```
+
+# SUPG advection operator example:
+```jldoctest
+# setup
+mesh = Mesh2D(1.0, 1.0);
+parameters = (wind = [1, 1]);
+mapping = nothing;
+collocate = false;
+supgadvection = GalleryOperator("supgadvection", 4, 4, mesh, parameters, collocatedquadrature = collocate, mapping = mapping);
+
+# verify
+println(supgadvection)
+
+# output
+
+finite element operator:
+2d mesh:
+    dx: 1.0
+    dy: 1.0
+
+2 inputs:
+operator field:
+  tensor product basis:
+    numbernodes1d: 4
+    numberquadraturepoints1d: 4
+    numbercomponents: 1
+    dimension: 2
+  evaluation modes:
+    interpolation
+    gradient
+operator field:
+  tensor product basis:
+    numbernodes1d: 4
+    numberquadraturepoints1d: 4
+    numbercomponents: 1
+    dimension: 2
+  evaluation mode:
+    quadratureweights
+
+1 output:
+operator field:
+  tensor product basis:
+    numbernodes1d: 4
+    numberquadraturepoints1d: 4
+    numbercomponents: 1
+    dimension: 2
+  evaluation mode:
+    gradient
+```
+
 """
 function GalleryOperator(
     name::String,
     numbernodes1d::Int,
     numberquadraturepoints1d::Int,
     mesh::Mesh;
+    parameters::Tuple{Float64,Float64},
```

Comment:
Can we make this a tuple of arbitrary length and contents??

Reply:
```suggestion
    parameters::Union{Tuple,Nothing} = nothing,
```
does this syntax work?